content/docs/start/data-management/pipelines.md

```diff
@@ -151,13 +151,12 @@ along with `git commit` to version DVC metafiles).
 
 [to remote storage]: /doc/start/data-and-model-versioning#storing-and-sharing
 
-## Dependency graphs (DAGs)
+## Dependency graphs (DAG)
 
-By using `dvc stage add` multiple times, and specifying <abbr>outputs</abbr> of
-a stage as <abbr>dependencies</abbr> of another one, we can describe a sequence
-of commands which gets to a desired result. This is what we call a _data
-pipeline_ or
-[_dependency graph_](https://en.wikipedia.org/wiki/Directed_acyclic_graph).
+By using `dvc stage add` multiple times to use <abbr>outputs</abbr> of a stage
+as <abbr>dependencies</abbr> of another one, we can describe a sequence of
+connected commands that gets to a desired result. This [dependency graph] (a
```

Comment:
It has gotten more formal and longer ... while we should try to do the opposite in get started ..

`connected` - doesn't provide enough value (it's clear more or less)

`cohesive pipeline` - is too fancy :)

etc ...

Reply:
Rolled back this change, for the most part.